src/main/java/com/rocketden/main/exception/TimerError.java

```diff
@@ -8,6 +8,7 @@
 @Getter
 public enum TimerError implements ApiError {
 
+    INVALID_DURATION(HttpStatus.BAD_REQUEST, "Please enter a valid duration between 1-60 minutes."),
```

Comment:
nit: Update this field if necessary depending on how you handle the 0-1 minute category.

Reply:
Ended up going with options 2 and 3, so this should be good unchanged I think